crashtracker/src/collector/crash_handler.rs

```diff
@@ -414,17 +540,21 @@ fn handle_posix_signal_impl(signum: i32, sig_info: *mut siginfo_t) -> anyhow::Re
 ///     will not yet be stored.  This would lead to unexpected behaviour for the
 ///     user.  This should only matter if something crashes concurrently with
 ///     this function executing.
-pub fn register_crash_handlers(create_alt_stack: bool) -> anyhow::Result<()> {
+pub fn register_crash_handlers() -> anyhow::Result<()> {
     if !OLD_HANDLERS.load(SeqCst).is_null() {
         return Ok(());
     }
 
+    let config_ptr = CONFIG.load(SeqCst);
+    anyhow::ensure!(!config_ptr.is_null(), "No crashtracking config");
+    let (config, _config_str) = unsafe { config_ptr.as_ref().context("config ptr")? };
```

Comment:
could this have a race condition if this is called twice?

Reply:
Or on the other side of a `fork`

Reply:
Lets add some tests for that (apparantly we already do in python, but not in libdatadog)

Reply:
Tried to clean this up, added a fork test.